ansible/deploy_commcarehq.yml

```diff
@@ -5,7 +5,7 @@
   # - celery
   # - proxy ?? maybe
   # - pillowtop
-  hosts: all
+  hosts: all:!riakcs:!stanchion
```

Comment:
if riakcs is deployed to a machine where we do need localsettings will this work? such as staging?